content/en/guide/v10/getting-started.md

```diff
@@ -32,11 +32,9 @@ Preact is packaged to be used directly in the browser, and doesn't require any b
 
 [🔨 Edit on Glitch](https://glitch.com/~preact-no-build-tools)
 
-The primary drawback of developing this way is the lack of JSX, which requires a build step. An ergonomic and performant alternative to JSX is documented in the next section.
```

Comment:
IMHO,
Not using JSX is not drawback.
That using `h` or `createElement` is difficult to write is drawback.